src/PointwiseFunctions/AnalyticSolutions/Burgers/Step.hpp

```diff
@@ -0,0 +1,95 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <limits>
+
+#include "DataStructures/DataBox/Prefixes.hpp"  // IWYU pragma: keep
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/Burgers/Tags.hpp"  // IWYU pragma: keep
+#include "Options/Options.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+class DataVector;
+// IWYU pragma: no_forward_declare Tensor
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace Burgers {
+namespace Solutions {
+/// \brief A propagating shock between two constant states
+///
+/// The shock propagates left-to-right, with profile
+/// \f$U(x, t) = U_R + (U_L - U_R) H(-(x - x_0 - v t))\f$.
+/// Here \f$U_L\f$ and \f$U_R\f$ are the left and right constant states,
+/// satisfying \f$U_L < U_R\f$; \f$H\f$ is the Heaviside function; \f$x_0\f$ is
+/// the initial (i.e., \f$t=0\f$) position of the shock; and
+/// \f$v = 0.5 (U_L + U_R)\f$ is the speed of the shock.
+///
+/// \note At the shock, where \f$x = x_0 + vt\f$, we have \f$U(x, t) = U_L\f$.
+/// (This is inherited from the Heaviside implementation `step_function`.)
+/// Additionally, the time derivative \f$\partial_t u0\f$ is zero, rather than
+/// the correct delta function.
+class Step {
+ public:
+  struct LeftValue {
+    using type = double;
+    static type lower_bound() { return 0.0; }
+    static type default_value() { return 1.0; }
```

Comment:
The defaults here and for RightValue give an inverted shock, which will fail the assertion in the constructor.

Reply:
Whoops, thanks for catching this